test/tests/meshgenerators/poly2tri_mesh_generator/poly2tri_with_holes.i

```diff
@@ -0,0 +1,35 @@
+[Mesh]
+  [outer_bdy]
+    type = PolyLineMeshGenerator
+    points = '-1.0 0.0 0.0
+              0.0 -1.0 0.0
+              1.0 0.0 0.0
+              0.0 2.0 0.0'
+    loop = true
+  []
+  [hole_1]
+    type = PolyLineMeshGenerator
+    points = '-0.5 -0.1 0.0
+              -0.3 -0.1 0.0
+              -0.3 0.1 0.0
+              -0.5 0.1 0.0'
+    loop = true
+  []
+  [hole_2]
+    type = PolyLineMeshGenerator
+    points = '0.3 -0.1 0.0
+              0.5 -0.1 0.0
+              0.5 0.1 0.0
+              0.3 0.1 0.0'
+    loop = true
+  []
+  [triang]
+    type = Poly2TriMeshGenerator
+    boundary = 'outer_bdy'
+    holes = 'hole_1
+             hole_2'
+    add_nodes_per_boundary_segment = 3
+    refine_boundary = false
+    desired_area = 0.05
```

Comment:
I did not run the case myself, just curious that will the final mesh contains the 1d elements from those `PolyLineMeshGenerator`s, also will we automatically have side sets corresponding to those three 1D polylines?

Reply:
It will, but by default just auto-assigned ids.  I'll add a test to cover the manually-assigned names case.

Reply:
Test will be going in with the next push.  And I just realized I misread your question - the final mesh will only have sidesets corresponding to those polylines.  With 2D hole meshes we can stitch the mesh in, but we don't have support for that with 1D hole meshes and it won't be trivial to add.